### PR TITLE
Add evlaluation workflow support

### PR DESCRIPTION
**Pull request by kannwism**

### CODE REVIEW

File: tests/integration/test_example_orders.py [hunk 1]
```diff
@@ -51,16 +51,22 @@ def test_quickstart(self):
         CLIENT_ID = os.getenv("CLIENT_ID")
         CLIENT_SECRET = os.getenv("CLIENT_SECRET")
         ENDPOINT = os.getenv("ENDPOINT")
+        CERT_PATH = os.getenv("CERT_PATH")
+        TOKEN_URL = os.getenv("TOKEN_URL")
```
Comment (Makuh17) [inline tests/integration/test_example_orders.py:55]: What is this now?

File: tests/integration/test_internal_orders.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+                truths=["Marsupial"],
+                metadata=[PromptMetadata(prompt="Hint: It has a pouch")],
+            )
+            .create()
+        )
+
+        order = (
+            self.rapi.new_order("Test Evaluation Order")
+            .workflow(EvaluationWorkflow(validation_set_id=validation_set.id))
+            .media([MediaAsset(path="examples/data/wallaby.jpg")])
```
Comment (Makuh17) [inline tests/integration/test_internal_orders.py:29]: Why do we need to add media here?

**Review by Makuh17 — COMMENTED** [review]
I am a little unsure about why we add media when creating the order?

File: tests/integration/test_internal_orders.py (hunk 2 above)
Comment (kannwism) [reply tests/integration/test_internal_orders.py]: Not necessary anymore

**Pull request merged**